returnn/search.py

```diff
@@ -32,50 +37,43 @@ def __init__(
         search_data,
         model_checkpoint,
         returnn_config,
+        returnn_python_exe,
+        returnn_root,
         *,
         output_mode="py",
         log_verbosity=3,
         device="gpu",
         time_rqmt=4,
         mem_rqmt=4,
         cpu_rqmt=2,
-        returnn_python_exe=None,
-        returnn_root=None,
     ):
         """
         :param dict[str] search_data: dataset used for search
         :param Checkpoint model_checkpoint:  TF model checkpoint. see `ReturnnTrainingJob`.
         :param ReturnnConfig returnn_config: object representing RETURNN config
+        :param tk.Path returnn_python_exe: path to the RETURNN executable (python binary or launch script)
+        :param tk.Path returnn_root: path to the RETURNN src folder
         :param str output_mode: "txt" or "py"
         :param int log_verbosity: RETURNN log verbosity
         :param str device: RETURNN device, cpu or gpu
         :param float|int time_rqmt: job time requirement in hours
         :param float|int mem_rqmt: job memory requirement in GB
         :param float|int cpu_rqmt: job cpu requirement in GB
```

Comment:
```suggestion
        :param float time_rqmt: job time requirement in hours
        :param float mem_rqmt: job memory requirement in GB
        :param int cpu_rqmt: job cpu requirement
```
in typing `float` also include all integer types, so `float|int` is redundant.